controllers/datadogagent/agent_test.go

```diff
@@ -3443,6 +3443,14 @@ func Test_newExtendedDaemonSetFromInstance_SecurityAgent_Runtime(t *testing.T) {
 			ReadOnly:  true,
 		},
 	}...)
+	securityAgentPodSpec.Containers[2].VolumeMounts = append(securityAgentPodSpec.Containers[2].VolumeMounts, []corev1.VolumeMount{
+		{
+			Name:      datadoghqv1alpha1.SecurityAgentRuntimePoliciesDirVolumeName,
+			MountPath: "/etc/datadog-agent/runtime-security.d",
```

Comment:
is it /opt ?
https://github.com/DataDog/datadog-operator/pull/342/files#diff-1e55b7aa7422a4461a33b141f491043abd3d73bf64215734d90850c38655e289R1638

Reply:
no here it is `/etc/` and `SecurityAgentRuntimePoliciesDirVolumeName` correspond to the emptyDir volume with aggregation of default files + custom file. (the magic appends inside an init container )